deployment/docker-compose-simulation-scenario2.yml

```diff
@@ -0,0 +1,72 @@
+version: "2"
+
+services:
+
+  # message bus
+  rabbit:
+    image: rabbitmq:management
+    container_name: rabbit
+    hostname: rabbit
+    ports:
+      - "8081:15672"
+      # Forwarding the port for testing
+      - "5672:5672"
+
+  mongodb:
+    image: mongo:4.0.0
+    volumes:
+      - ./data/mongodb:/data
+    ports:
+      - "27017:27017"
+
+  frontier:
+    image: squirrel.frontier:latest
```

Comment:
Please update the Docker images in all files.

Reply:
i missed that one, fixed